WordPress/Classes/ViewRelated/Tags/EditTagView.swift

```diff
@@ -0,0 +1,215 @@
+import SwiftUI
+import WordPressUI
+import WordPressKit
+import WordPressData
+import SVProgressHUD
+
+struct EditTagView: View {
+    @Environment(\.dismiss) private var dismiss
+    @StateObject private var viewModel: EditTagViewModel
+
+    init(tag: RemotePostTag?, tagsService: TagsService) {
+        self._viewModel = StateObject(wrappedValue: EditTagViewModel(tag: tag, tagsService: tagsService))
```

Comment:
This is fine 👍  Basically it's only unsafe if the view retains its identity. A good test would be – if it has a `@State` property, would it retain its state after I init the view again in the place in the view hierarchy. If the answer is no, you can use `StateObject(wrappedValue` safely.